src/App.tsx

```diff
@@ -15,35 +17,97 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+const getPreparedGoods = (sortField: SortType, isReverse: boolean) => {
+  const copyOfGoods = [...goodsFromServer];
+
+  if (sortField) {
+    copyOfGoods.sort((value1, value2) => {
+      switch (sortField) {
+        case SortType.alphabetically:
+          return value1.localeCompare(value2);
+        case SortType.length:
+          return value1.length - value2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReverse) {
+    copyOfGoods.reverse();
+  }
+
+  return copyOfGoods;
+};
+
 export const App: React.FC = () => {
+  const [isReverse, setIsReverse] = useState(false);
+  const [sortField, setSorfield] = useState<SortType>(SortType.default);
```

Comment:
The state setter function name `setSorfield` seems to be a typo. Consider renaming it to `setSortField` to match the naming convention and improve readability. [CODE STYLE]